README.md

```diff
@@ -577,7 +577,7 @@ $ cd RxJava/
 $ ./gradlew build
 ```
 
-Further details on building can be found on the [Getting Started](https://github.com/ReactiveX/RxJava/wiki/Getting-Started) page of the wiki.
+Further details on the building can be found on the [Getting Started](https://github.com/ReactiveX/RxJava/wiki/Getting-Started) page of the wiki.
```

Comment:
I think "the building" would mean like "the house" instead of constructing the library binaries. Keep as it was.